deploy/cloud-ingress-operator-configuration/apischeme/config.yaml

```diff
@@ -7,6 +7,9 @@ selectorSyncSet:
   - key: api.openshift.com/sts
     operator: NotIn
     values: ["true"]
+  - key: api.openshift.com/wif
+    operator: NotIn
+    values: ["true"]
```

Comment:
Does this actually stop the operator from deploying? I would think this only removes configuration. Is this good enough? 

IIRC it's also deployed to AWS STS, it just doesn't do anything. If that's the case, I think it's fine.

Reply:
We will need to update [https://github.com/openshift/cloud-ingress-operator/blob/03bb9046ea211f56b782012100b0c72d58a1418f/hack/olm-registry/olm-artifacts-template.yaml#[…]5](https://github.com/openshift/cloud-ingress-operator/blob/03bb9046ea211f56b782012100b0c72d58a1418f/hack/olm-registry/olm-artifacts-template.yaml#L37-L45) to not deploy the CIO operator itself on WIF clusters (and similar for MVO)

Reply:
Hello Michael is right, I need to update the CIO repo as well
This won't be enough

Reply:
FYI, CIO PR:
https://github.com/openshift/cloud-ingress-operator/pull/340